docs/examples/fork-process-model/flask-uwsgi/app.py

```diff
@@ -0,0 +1,57 @@
+import flask
```

Comment:
License is missing from several files.

Reply:
Added, I didn't know example also require License.